src/pages/SubmissionDetails.js

```diff
@@ -36,34 +51,90 @@ export const SubmissionDetails = ({ submissionId }) => {
+      }
+    }
+
+    const loadMethodsData = async () => {
+      try {
+        const methodsResponse = await Ajax(signal).Cbas.methods.get()
+        const allMethods = methodsResponse.methods
+        setMethodsData(allMethods)
+      } catch (error) {
+        notify('error', 'Error loading methods data', { detail: await (error instanceof Response ? error.text() : error) })
+      }
+    }
+
     loadRunsData()
+    loadRunSetData()
+    loadMethodsData()
   })
 
+  const specifyRunSet = _.filter(r => r.run_set_id === submissionId, runSetData)
+  const methodId = specifyRunSet[0]?.method_id
+  const getSpecificMethod = _.filter(m => m.method_id === methodId, methodsData)
   const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, runsData)
 
   const firstPageIndex = (pageNumber - 1) * itemsPerPage
   const lastPageIndex = firstPageIndex + itemsPerPage
   const paginatedPreviousRuns = sortedPreviousRuns.slice(firstPageIndex, lastPageIndex)
 
   const rowWidth = 100
-  const rowHeight = 200
+  const rowHeight = 50
   return h(Fragment, [
-    headerBar(),
-    div({ style: { margin: '4em' } }, [
-      div({ style: { display: 'flex', marginTop: '0.5rem', justifyContent: 'space-between' } }, [
+    div({
+      style: {
+        borderBottom: '2px solid rgb(116, 174, 67)',
+        boxShadow: 'rgb(0 0 0 / 26%) 0px 2px 5px 0px, rgb(0 0 0 / 16%) 0px 2px 10px 0px',
+        position: 'relative'
+      }
+    }, [
+      headerBar(),
+      div({ style: { marginLeft: '4em', display: 'flex', marginTop: '0.5rem', justifyContent: 'space-between' } }, [
         h1(['Submission Details']),
         h(ButtonOutline, {
+          style: { margin: '2em' },
           onClick: () => Nav.goToPath('root')
         }, ['Submit a new workflow'])
       ]),
-      div([
-        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`])
-      ]),
+      div({ style: { marginLeft: '4em' } }, [
+        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`]),
+        h2(['workflow: ', getSpecificMethod[0]?.name]),
+        h3(['Submission date: ', specifyRunSet[0] && Utils.makeCompleteDate(specifyRunSet[0].submission_timestamp)]),
+        h3(['Duration: ', specifyRunSet[0] && Utils.customFormatDuration(duration(specifyRunSet[0]))]),
+        h3(['Submitted by: Batch Teams (HARDCODED)'])
+      ])
+    ]),
+    div({
+      style: {
+        backgroundColor: 'rgb(235, 236, 238)',
+        display: 'flex',
+        flex: '1 1 auto',
+        flexDirection: 'column',
+        padding: '1rem 3rem'
+      }
+    }, [
       div({
         style: {
           marginTop: '1em', height: tableHeight({ actualRows: paginatedPreviousRuns.length, maxRows: 12.5, heightPerRow: 250 }), minHeight: '10em'
         }
       }, [
+        div([h2(['Workflows'])]),
+        div([h3(['Filter by: '])]),
```

Comment:
When I tried this, this drop down list was empty. 
![image](https://user-images.githubusercontent.com/13006282/207161317-7817dc38-3dc3-420f-986f-d216e8433eb5.png)

Reply:
If that's deliberate, is there a Jira ticket for adding options?